### DISCUSSION TITLE
How can I put Copilot chat back in the main sidebar?

### DISCUSSION THREAD

**Discussion by bengosney** [community/community#143300] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

As the title says I'd like to put Copilot chat back in the main sidebar. I have limited screen space and having both sidebars open leaves less than 80 chars to actually write code in and closing the main sidebar is an unnecessary step.

**Comment by sunzunyi** [comment, marked as answer]
Hey, the link below should help solve your problem.
[https://stackoverflow.com/questions/79162102/how-can-i-put-copilot-chat-back-in-the-main-sidebar]

**Comment by bengosney** [comment]
For anyone who doesn't want to follow a link, you can just drag and drop the icon from the secondary sidebar to the primary
